www/topical-software.rst

```diff
@@ -90,6 +90,7 @@ Working environments
 - `IPython <http://ipython.org>`__: an interactive environment with many features geared towards efficient work in typical scientific usage. It borrows many ideas from the interactive shells of Mathematica, IDL, Matlab and similar packages. It includes special support for the matplotlib and gnuplot plotting packages. IPython also has support for (X)Emacs, to be used as a full IDE with IPython as the interactive Python shell.
 - `Jupyter Notebook <http://jupyter.org/>`__: a web application that works off of the IPython and allows you to create and share documents containing live code and can be used in all sorts of contexts from statistical modeling to data cleaning.
 - `Spyder <https://pythonhosted.org/spyder/>`__: A Qt based IDE suited to developing scientific applications.  Includes integrated and external Python consoles, code checking built into the editor, a graphical class browser and full support for matplotlib graphs.
+- `Microsoft Visual Studio <https://www.visualstudio.com/>`__: A free, rich IDE that now natively supports Python and Anaconda. CPython, IronPython, the IPython REPL, debugging, profiling, Git and GitHub are also supported. Also has a lighter version, named Visual Studio Code, which is a code editor with debugger. Both feature powerful IntelliSense, and support Windows and OS X, plus Linux in case of Code.
```

Comment:
"that now natively" -> "that natively"  (this text probably won't be updated for a while)

"A, B, C, and D are also supported" -> "It supports A, B, C, ... and D."

"Git and GitHub" -> "Git, and GitHub" (I prefer the Oxford comma, not a big deal though)

The next two sentences could be condensed a bit:
"Visual Studio supports Windows and macOS, and the lighter weight Visual Studio Code also supports Linux. Both feature editing, debugging, and IntelliSense capabilities for Python."

Reply:
About "now", I wanted to stress that, but okay, I agree.
I think comma is not needed when we put "and" at the end.
I wouldn't change the rest, because I wanted to stress what Code is. It's equivalent, I guess it's just a different angle. But, others can jump in freely.
I had put macOS first, because that's how Microsoft has put it on their website. Now, when I look at it, Apple themselves call it like that: https://www.apple.com/macos/sierra/
So, I guess it is macOS, not OS X.

Reply:
> So, I guess it is macOS, not OS X.

Looks like Apple rebranded it last year. sigh....